### ISSUE TITLE
vdiff model is no longer available

### ISSUE THREAD

**Issue by graytFox** [pixray/pixray#63]
Hello! It looks like the model for vdiff isn't available at the URL currently in Pixray anymore. Attempting to use the vdiff drawer gives me this:

```
(base) fox@fox-MS-7B98:~/pixray$ python pixray.py --drawer=vdiff --prompts="an excessively fuzzy panda" --outdir panda
Running with 30x1 = 30 cuts
Using seed: 7387649654636579532
Downloading models/yfcc_2.pth from https://v-diffusion.s3.us-west-2.amazonaws.com/yfcc_2.pth, please wait
--2022-05-04 13:39:59--  https://v-diffusion.s3.us-west-2.amazonaws.com/yfcc_2.pth
Resolving v-diffusion.s3.us-west-2.amazonaws.com (v-diffusion.s3.us-west-2.amazonaws.com)... 52.218.228.113
Connecting to v-diffusion.s3.us-west-2.amazonaws.com (v-diffusion.s3.us-west-2.amazonaws.com)|52.218.228.113|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-05-04 13:39:59 ERROR 404: Not Found.

Ignoring non-zero exit:  b''
Traceback (most recent call last):
  File "/home/fox/pixray/pixray.py", line 2135, in <module>
    main()
  File "/home/fox/pixray/pixray.py", line 2129, in main
    do_init(settings)
  File "/home/fox/pixray/pixray.py", line 613, in do_init
    drawer.load_model(args, device)
  File "/home/fox/pixray/vdiff.py", line 85, in load_model
    model.load_state_dict(torch.load(checkpoint, map_location='cpu'))
  File "/home/fox/miniconda3/lib/python3.9/site-packages/torch/serialization.py", line 608, in load
    return _legacy_load(opened_file, map_location, pickle_module, **pickle_load_args)
  File "/home/fox/miniconda3/lib/python3.9/site-packages/torch/serialization.py", line 777, in _legacy_load
    magic_number = pickle_module.load(f, **pickle_load_args)
EOFError: Ran out of input
```

And spits out a 0 byte `vfcc_2.pth` file in the models folder. Attempting to follow the link it gives also yields a 404 page saying the bucket no longer exists.

**Comment by ulolol** [comment]
Hi,
I can confirm facing the same issues as @graytFox above. Would love to get any help regarding this.
Thanks!

**Comment by LucienGab** [comment]
I'm having the same issue

**Comment by samedii** [comment]
The models have been moved and can be found here now:
```
model_urls = {
    "yfcc_2": "https://the-eye.eu/public/AI/models/v-diffusion/yfcc_2.pth",
    "yfcc_1": "https://the-eye.eu/public/AI/models/v-diffusion/yfcc_1.pth",
    "cc12m_1": "https://the-eye.eu/public/AI/models/v-diffusion/cc12m_1.pth",
    "cc12m_1_cfg": "https://the-eye.eu/public/AI/models/v-diffusion/cc12m_1_cfg.pth",
    "wikiart": "https://the-eye.eu/public/AI/models/v-diffusion/wikiart_256.pth",
}
```

**Comment by dribnet** [comment]
Thanks - this is now fixed in master and should be part of the next release.

**Comment by dribnet** [comment]
This has now been released in 1.9.1 - thanks for this report and tracking down the solution.
